src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdater.java

```diff
@@ -56,14 +56,15 @@ public JsonNode updateCase(CcdCallbackDto ccdCallbackDto) {
             });
 
         CcdBundleDTO bundle = bundleFactory.create(configuration, ccdCallbackDto.getCaseData());
-        bundles.add(bundleDtoToBundleJson(bundle));
 
         automatedStitchingExecutor.startStitching(
                 ccdCallbackDto.getCaseId(),
                 "asyncStitchingComplete",
                 ccdCallbackDto.getJwt(),
                 bundle);
 
+        bundles.add(bundleDtoToBundleJson(bundle));
```

Comment:
I'm not sure I understand this - why add a bundle after stitching has started?

Reply:
startStitching updates the status